# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cinepaint/gcc44.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Index: cinepaint/lib/player.c
===================================================================
RCS file: /cvsroot/cinepaint/cinepaint-project/cinepaint/lib/player.c,v
retrieving revision 1.5
diff -u -r1.5 player.c
--- cinepaint/lib/player.c	21 Feb 2006 08:24:05 -0000	1.5
+++ cinepaint/lib/player.c	18 Nov 2009 21:14:06 -0000
@@ -474,7 +474,7 @@
 
 void
 gimp_layer_set_name (gint32  layer_ID,
-		     char   *name)
+		     const char   *name)
 {
   GParam *return_vals;
   int nreturn_vals;
Index: cinepaint/lib/plugin_main.h
===================================================================
RCS file: /cvsroot/cinepaint/cinepaint-project/cinepaint/lib/plugin_main.h,v
retrieving revision 1.11
diff -u -r1.11 plugin_main.h
--- cinepaint/lib/plugin_main.h	25 Oct 2007 10:17:36 -0000	1.11
+++ cinepaint/lib/plugin_main.h	18 Nov 2009 21:14:07 -0000
@@ -402,7 +402,7 @@
 DLL_API void          gimp_layer_set_mode                  (gint32        layer_ID,
 						    GLayerMode    mode);
 DLL_API void          gimp_layer_set_name                  (gint32        layer_ID,
-						    char         *name);
+						    const char         *name);
 DLL_API void          gimp_layer_set_offsets               (gint32        layer_ID,
 						    gint          offset_x,
 						    gint          offset_y);
